packages/core/src/discovery.ts

```diff
@@ -0,0 +1,33 @@
+export type WalletProvider = {
+  id: string
+  name: string
+  icon: string
+  downloads:
+    | { chrome?: `https://chrome.google.com/webstore/detail/${string}` }
+    | { firefox?: `https://addons.mozilla.org/en-US/firefox/addon/${string}` }
+}
+
+const wallets: WalletProvider[] = [
```

Comment:
order changed? :)
I'm fine with it as long as we drop the "community" sort order and randomize the discovery list by default (default sort -> random).

Reply:
removed community, random is default, or explicit